src/test/functional/pages/BuzzPage.ts

```diff
@@ -1,32 +1,21 @@
 import { BasePage } from "./BasePage";
-import { Locator, Page } from "@playwright/test";
 export class BuzzPage extends BasePage {
-    readonly sharePhotosButton: Locator;
-    readonly fileInput: Locator;
-    readonly submitPhoto: Locator
-    readonly threeDotsIcon: Locator
-    readonly deletePostParagraph: Locator
-    readonly confirmDeleteButton: Locator
-    readonly photoImg: Locator
-    readonly noPostParagraph: Locator
-    readonly buzzPageButton: Locator
-  
-    constructor(page: Page) {
-      super(page)
-      this.sharePhotosButton = page.locator('button:has-text("Share Photos")');
-      this.fileInput = page.locator('input[type="file"]');
-      this.submitPhoto = page.locator('button.oxd-button--main:has-text("Share")');
-      this.threeDotsIcon = page.locator('i.oxd-icon.bi-three-dots');
-      this.deletePostParagraph = page.locator('p:has-text("Delete Post")');
-      this.confirmDeleteButton = page.locator('button:has-text("Yes, Delete")');
-      this.photoImg = page.locator('.orangehrm-buzz-photos-item img')
-      this.noPostParagraph = page.locator('p:has-text("No Posts Available")')
-      this.buzzPageButton = page.locator('span:has-text("Buzz")')
-    }
-
-    async sharePhotos(): Promise<void> {
-    const filePath = '../test/functional/files/file1.png';
-        await this.sharePhotosButton.click();
+    public readonly sharePhotoButton = this.page.getByRole("button", { name: "Share Photos" })
+    public readonly fileInput = this.page.locator('input[type="file"]');
+    public readonly submitPhoto = this.page.locator('button.oxd-button--main:has-text("Share")');
+    public readonly threeDotsIcon = this.page.locator('i.oxd-icon.bi-three-dots');
+    public readonly deletePostParagraph = this.page.getByText("Delete Post"); 
+    public readonly editPostParagraph = this.page.getByText("Edit Post");
+    public readonly confirmDeleteButton = this.page.getByRole("button", { name: "Yes, Delete" })
+    public readonly photoImg = this.page.locator('.orangehrm-buzz-photos-item img')
+    public readonly noPostParagraph = this.page.locator('p:has-text("No Posts Available")')
+    public readonly buzzPageButton = this.page.locator('span:has-text("Buzz")')
+    public readonly textFieldInPostEdit = this.page.locator('.orangehrm-buzz-post-modal-header-text .oxd-buzz-post .oxd-buzz-post-input')
+    public readonly removePictureButton = this.page.locator('.oxd-icon-button.orangehrm-photo-input-remove i.bi-x')
+    public readonly confirmEditedPost = this.page.locator('.orangehrm-buzz-post-modal-actions button')
+    
+    async sharePhotos(filePath:string): Promise<void> {
```

Comment:
Please use proper access in all of your methods and fields in all of your classes.